config/routechangerconfig.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+type routeChangerConfig struct {
+	Reg  *regexp.Regexp
+	Repl []byte
+}
+
+func (rcc routeChangerConfig) String() string {
+	if rcc.Reg == nil {
+		return ""
+	}
+	return fmt.Sprintf("/%s/%s/", rcc.Reg, rcc.Repl)
+}
+
+func (rcc *routeChangerConfig) Set(value string) error {
+	a := strings.Split(value, "/")
+	if len(a) != 4 {
+		return fmt.Errorf("unexpected size of string split: %d", len(a))
+	}
+	reg, repl := a[1], a[2]
+	rcc.Reg = regexp.MustCompile(reg)
```

Comment:
let's not use MustCompile here. We can use just compile and return an error. Then the flag package can display the error with the right context, and there is no value in showing the stacktrace here

Reply:
changed

Reply:
good catch